### ISSUE TITLE
MBP: User sign in Page gives "Page doesn't exist error"

### ISSUE THREAD

**Issue by WaBirder57** [washingtontrails/vms#484]
In certain situations when a user logs into MyBackpack and click Sign In they are logged in but are directed to a bad URL - https://www.wta.org/http%3A//www.wta.org/%40%40backpack   This gives them the "This page does not seem to exist…" web page.

Here is the URL for the login screen for my recent test of this:

https://www.wta.org/acl_users/credentials_cookie_auth/require_login?came_from=http%253A//www.wta.org/%2540%2540backpack

Here is button that I click to SignIn on that page.:

![mbpsigninscreen](https://cloud.githubusercontent.com/assets/8312206/11697124/02b0906a-9e6d-11e5-849e-36cadf5dd552.JPG)

This has existed prior to the MBP launch but with more users using MyBackpack we should get this fixed.

**Comment by davisagli** [comment]
Can you show me a screenshot of where you clicked that took you to the login form?

**Comment by WaBirder57** [comment]
I'm not able to reproduce this error now - it only crops up from time to time.  

I get to Sign In page by going from home page to the My Backpack menu, drop down to Login.  I'm using Firefox and this happened the first time I tried to access today (perhaps login cookie timed out overnight).  Now that I have logged in and out I'm not able to reproduce.  

I can work with a user who has reported this a bug to get their steps to reproduce this error.

**Comment by WaBirder57** [comment]
Here is the bug reported by the user:

Win 8.1, IE 11.0   OR   Win 8.1, Chrome 47.0
If I’m logged out, click “My Backpack”, and then enter my email and password, I get “This page does not seem to exist...” (https://www.wta.org/http%3A//www.wta.org/%40%40backpack). It works fine if I click on “Log in” from the logged out state. I have the same problem when clicking “Write a trip report” from the logged out state. This is a huge bug – it’s terrible when the first thing the user sees after logging in (however they do it) is an error message.

**Comment by davisagli** [comment]
I'm able to reproduce the issue. (I think it may only be a problem when starting on http, not https)

**Comment by davisagli** [comment]
Yeah, it was a problem with the redirect from http to https. Fixed.
